### PR TITLE
Update description of `upstream_ref` and fix typo

### PR DESCRIPTION
**Pull request by mfocko**
- Document new implementation of `upstream_ref` that allows globbing pattern
- Fix typo in default command for `current_version_command`

Merge after packit/packit#960

### CODE REVIEW

File: content/docs/configuration.md [hunk 1]
```diff
@@ -58,6 +58,9 @@ and config file path are set every time by default).
 #### upstream_ref
 
 (*string*) Git reference to last upstream git commit (for source-git repos).
+Can be set to *fixed* commit hash, tag or a branch. You can also use globbing pattern
```
Comment (csomh) [inline content/docs/configuration.md:61]: All commit hashes are *fixed*, aren't they? :smiley: 

```suggestion
Can be set to commit hash, tag or a branch name. You can also use globbing pattern
```

File: content/docs/configuration.md [hunk 2]
```diff
@@ -58,6 +58,9 @@ and config file path are set every time by default).
...
+to find a tag. In case you want to use globbing pattern for a branch, prefix the
+pattern with `branches/`, e.g. for a branch matching `*-release` set to `branches/*-release`.
```
Comment (csomh) [inline content/docs/configuration.md:63]: I :heart: examples! Thank you!

**Review by csomh — COMMENTED** [review]
Nice!

**Review by TomasTomecek — APPROVED** [review]

File: content/docs/configuration.md (hunk 1 above)
Comment (mfocko) [reply content/docs/configuration.md:61]: Seemed like a good word ordering 🤔 anyways, it was meant to emphasize fixed branch (like `stable`) or tag... maybe replace with `specific`? or remove like you suggested

**Pull request merged**